.changelog/unreleased/features/9680-config-bootstrap-ppers.md

```diff
@@ -0,0 +1,3 @@
+- `[config]` Introduce `BootstrapPeers` to the config to allow
```

Comment:
```suggestion
- `[config]` Introduce `p2p.bootstrap_peers` to the config to allow
```

Operators don't see the `BootstrapPeers` variable - they see the configuration file entry.